ocs_ci/templates/app-pods/fedora_dc_with_linuxtarfiles.yaml

```diff
@@ -0,0 +1,45 @@
+kind: DeploymentConfig
```

Comment:
we already have a fedora yaml similar to this and has a fixture for it. please combine these